samples/04_gis_analysts_data_scientists/classifying_human_activity_using _tabPFN_classifier.ipynb

```diff
@@ -0,0 +1,2179 @@
+{
```

Comment:
<p>Once the explanatory variables&nbsp;<code>X</code>&nbsp;are defined, they are used as input in the&nbsp;<code>prepare_tabulardata</code>&nbsp;method from the tabular learner in <code>arcgis.learn</code>. The method takes the feature layer or a spatial dataframe containing the dataset and prepares it for fitting the model.</p><p>The input parameters required for the tool are used as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>